cmd/av/main.go

```diff
@@ -149,6 +152,22 @@ func checkCliVersion() {
 var once sync.Once
 var lazyGithubClient *gh.Client
 
+func discoverGitHubAPIToken() string {
```

Comment:
Probably just makes sense to call this as part of `getClient` and return the "github token must be set" error from `getClient`.

Reply:
Done.